tests/index.test.ts

```diff
@@ -0,0 +1,291 @@
+import { GradeLetter, Severity } from "../src/constants/enums";
```

Comment:
File was moved from src/ to tests/ to not show up in the build. Also had a few tests added